$NetBSD: patch-Cargo.toml,v 1.1 2026/05/22 06:40:25 wiz Exp $

===> Building for greed-5.1
error: unexpected end of input while parsing minor version number
 --> Cargo.toml:3:11
  |
3 | version = "5.1"
  |           ^^^^^
*** Error code 101


https://doc.rust-lang.org/cargo/reference/manifest.html#the-version-field


--- Cargo.toml.orig	2026-05-22 06:36:44.445007403 +0000
+++ Cargo.toml
@@ -1,6 +1,6 @@ name = "greed"
 [package]
 name = "greed"
-version = "5.1"
+version = "5.1.0"
 edition = "2021"
 license = "BSD-2-Clause"
 
